surefire-integration-tests/src/test/java/org/apache/maven/surefire/its/Java9FullApiIT.java

```diff
@@ -78,12 +83,13 @@ public void shouldLoadMultipleJavaModules_ToolchainsXML() throws IOException
                                             .addGoal( "--toolchains" )
                                             .addGoal( System.getProperty( "maven.toolchains.file" ) )
                                             .execute( "verify" )
-                                            .verifyErrorFree( 2 );
+                                            .verifyErrorFree( 1 );
```

Comment:
Why it has changed from 2 to 1?
There was two tests?

Reply:
I have dropped tet_corba_mod.
It is not possible to have  legacy "corba module" + dependency of JTA, because module java.corba require legacy transaction packages which be loaded from the JDK and so in turn this will hide the explicit dependency.

I could create one more IT only for corba

Reply:
@Tibor17 do you think it has some value to have an IT for corba ? I think that the given one is already a good proof, it tests other more important modules.
Tell me, if you want I will clone this IT test only of corba